pkg/operation/cloudbotanist/openstackbotanist/infrastructure.go

```diff
@@ -125,8 +122,8 @@ func (b *OpenStackBotanist) generateTerraformBackupConfig() map[string]interface
 			"region":     b.Seed.Info.Spec.Cloud.Region,
 		},
 		"container": map[string]interface{}{
-			"name": fmt.Sprintf("%s-%s", b.Shoot.SeedNamespace, utils.ComputeSHA1Hex([]byte(b.Shoot.Info.Status.UID))[:5]),
+			"name": b.Operation.BackupInfrastructure.Name,
```

Comment:
What will happen to the existing buckets? Will they be recreated? How does Terraform behave?